server/.env

```diff
@@ -1,2 +1,10 @@
 TEAM_NAME="Shums, Kareem, Gabriel, Moshe"
 FLASK_ENV="development"
+
+MOSHES_GOOGLE_API_KEY = 'AIzaSyAN3q6E8D80h38QQsiSWQ5keW6YSRMPqYc'
+
+# Database variables
+DB_USER = 'postgres'
+DB_PASSWORD = '123456'
+DB_URL = 'localhost:5432'
+DB_NAME = 'team-bluejay'
```

Comment:
this file shouldn't be in git commit.  How do I remove it from git commit. Can I just delete file?